cloud-deployment-guide/editor-and-features.md

```diff
@@ -18,12 +18,22 @@ All Tiny Cloud accounts include a free, pre-configured cloud image proxy service
 
 Insert the TinyMCE editor code into the application to load TinyMCE for the first time. The code inserting is done by adding the following script tag into your app, or in the case of a web page the `<head>` of the page:
 
-The following example adds a script tag into the application that inserts the code. Use the `<head>` of the page on a web page.
+The following example adds a script tag into the application that inserts the code. Use the `<head>` of the page on a web page. Be sure to substitute 'no-api-key' with your api key.
 
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
 ```
 
+||Value|Description|
+|---|---|---|
+|1|no-api-key|Replace with your api key|
+|2|origin=|A `referrerpolicy` specifies how much of the current page's URL is sent in the `Referer` header when the browser fetches page resources (for example, the TinyMCE editor). The use of the `Referer` header ensures API keys are only used on domains registered to their owners. We only care about the domain portion however (similar to the operation of `Origin` header), so for improved performance and privacy always set the `referrerpolicy` to `origin` when requesting Tiny resources.|
```

Comment:
Couple minor typos with "Referrer" here by the looks?

Reply:
Referer is the official (incorrect) spelling. ReferrerPolicy chose not to preserve that misspelling.

Reply:
> Note that Referer is actually a misspelling of the word "referrer". The Referrer-Policy header does not share this misspelling.

(Source: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)

Reply:
I'd never noticed that, but good to know :+1: